static-pages/for-educators.md

```diff
@@ -1,2 +1,27 @@
-title: For Educators 
+title: Make your Mark - Educators Resources
```

Comment:
Not sure whether this should be "Educators Resources" or "Educator's Resources" or "Educators' Resources" or maybe "Resources for Educators"